rgw/v2/tests/s3_swift/configs/test_lc_rule_reverse_transition.yaml

```diff
@@ -0,0 +1,60 @@
+#test_bucket_lifecycle_object_expiration_transition.py
+#polarion-id: CEPH-83573373
+config:
+  user_count: 1
+  bucket_count: 1
+  objects_count: 10
+  parallel_lc: False
+  test_lc_transition: True
+  pool_name: data.cold
+  storage_class: cold
+  ec_pool_transition: False
+  multiple_transitions: True
+  two_pool_transition: True
+  second_pool_name: data.glacier
+  second_storage_class: glacier
+  rgw_lc_debug_interval: 40
```

Comment:
@hmaheswa can we reduce the lc debug interval back to 30

Reply:
done @viduship 
updated pass log: http://magna002.ceph.redhat.com/ceph-qe-logs/Hemanth_Sai/PR_lc_reverse_transition/reef_lc_debug_30_test_lc_rule_reverse_transition.console.log